tests/e2e/containers/config.go

```diff
@@ -27,10 +27,10 @@ const (
 	previousVersionOsmoTag        = "v15.x-9fa047c2-1687827963"
 	// Pre-upgrade repo/tag for osmosis initialization (this should be one version below upgradeVersion)
 	previousVersionInitRepository = "osmolabs/osmosis-e2e-init-chain"
-	previousVersionInitTag        = "v15.x-9fa047c2-1687827963"
+	previousVersionInitTag        = "v15-fast-vote"
```

Comment:
I need to wait till this PR gets merged before adding the vote speedup to v15x, otherwise I will break CI

Reply:
Vote speed up involves reduces the vote time which is now possible due to all validators voting concurrently. We would otherwise be waiting multiple blocks for confirmation despite all validators having completed their votes